tests/test_model.py

```diff
@@ -0,0 +1,18 @@
+from app.models import User
+from model_base import ModelBase
+
+class UserModelTestCase(ModelBase):
+
+    def test_ld_user_hash(self):
+        u = User(email = 'test@example.com')
+        emailHash = u.get_email_hash()
+        userEmailHash = u.get_ld_user()['key']
+
+        self.assertEqual(userEmailHash, emailHash)
+
+    def test_ld_random_user(self):
+        u = User(email = 'test@example.com')
+        emailHash = u.get_email_hash()
+        userEmailHash = u.get_ld_user(random=True)['key']
```

Comment:
@manuelPartida see this test case, if I add the argument `random=True` it will make a random UUID rather than using the `get_email_hash()` method. If you pass this argument into `get_ld_user()` whenever you want to have a new random user then this will allow you to easily get enough unique views on the dashboard to start running AB tests.

Reply:
@levlaz Awesome, thanks for the example! Ill be sure to pass the argument when i create the route for the experimentation page.